planemo/galaxy/profiles.py

```diff
@@ -108,16 +130,66 @@ def ensure_profile(ctx, profile_name, **kwds):
     return _profile_options(ctx, profile_name, **kwds)
 
 
+def create_alias(ctx, alias, obj, profile_name, **kwds):
+    profile_options, profile_options_path = _load_profile_to_json(ctx, profile_name)
+    
+    if profile_options.get('aliases'):
+        profile_options['aliases'][alias] = obj
+    else:  # no aliases yet defined
+        profile_options['aliases'] = {alias: obj}
+    
+    with open(profile_options_path, 'w') as f:
+        json.dump(profile_options, f)
+
+    return 0
+
+
+def list_alias(ctx, profile_name, **kwds):
+    profile_options, _ = _load_profile_to_json(ctx, profile_name)
+    return profile_options.get('aliases', {})
+
+
+def delete_alias(ctx, alias, profile_name, **kwds):
+    profile_options, profile_options_path = _load_profile_to_json(ctx, profile_name)
+    if alias not in profile_options.get('aliases', {}):
+        return 1
+    else:
+        del profile_options['aliases'][alias]
+
+    with open(profile_options_path, 'w') as f:
+        json.dump(profile_options, f)
+
+    return 0
+
+
+def translate_alias(ctx, alias, profile_name):
+    if not profile_name:
+        return alias
+    # print(_load_profile_to_json(ctx, profile_name))
+    aliases = _load_profile_to_json(ctx, profile_name)[0].get('aliases', {})
```

Comment:
Maybe stick the aliases in their own dictionary inside the main dictionary instead of risking clashes with option names?

Reply:
Nevermind - you're doing this. Sorry for the noise.